Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthRequestParams.kt

```diff
@@ -0,0 +1,12 @@
+package com.braintreepayments.api.venmo
+
+import com.braintreepayments.api.BrowserSwitchOptions
+
+/**
+ * Used to request Venmo authentication via [VenmoLauncher.launch] )}
+ */
+data class VenmoPaymentAuthRequestParams internal constructor(
+    // TODO: this should be internal and restricted to library scope but can't be until
+    //  VenmoClientUnitTest is converted to Kotlin
```

Comment:
@tdchow I think ideally all the request params class would follow this structure and have the `browserSwitchOptions` param internal with library visibility. This breaks the unit tests, so we can't do that unless also converting the `VenmoClientUnitTest` class. Can I leave this as is for your ticket to unify all the auth request classes and then we can decide if it's worth converting/updating?

Reply:
Gotcha. Yeah I can take a look as a part of the the other interface story.